compiler/src/iree/compiler/Dialect/Flow/Transforms/FormDispatchRegions.cpp

```diff
@@ -556,8 +593,7 @@ static bool isFusableWithProducer(OpOperand &operand, bool aggressiveFusion) {
     return false;
   }
 
-  return areOpsAggresiveFusable(producer, consumer,
-                                /*allowConsumerParallelismPessimization=*/false,
```

Comment:
I didn't remember the reason we disabled consumer parallelism pessimization, do we want to drop that constraint here?

Reply:
That was being conservative. We are dropping that now. Also the `allowConsumerParallelismPessimization` was a bit of a misnomer... now the same logic is used to check for "fusability" whether we are fusing with producer or consumer.